backends-common/redis/src/main/java/org/apache/james/backends/redis/RedisConfiguration.scala

```diff
@@ -158,4 +177,114 @@ case class ClusterRedisConfiguration(redisURI: RedisUris, ioThreads: Option[Int]
     .add("redis.ioThreads", ioThreads)
     .add("redis.workerThreads", workerThreads)
     .toString
+}
+
+object SentinelRedisConfiguration {
```

Comment:
It would make more sense to split this work into two PRs:
- Sentinel redis configuration
- Redis SSL

Actually the redis sentinel configuration should be an all other ticket

Reply:
created another PR for sentinel https://github.com/apache/james-project/pull/2420